.dev-lib

```diff
@@ -0,0 +1,8 @@
+DEFAULT_BASE_BRANCH=develop
+PHPCS_GIT_TREE=master
+PHPCS_IGNORE='vendor/*,tests/*,dev-lib/*,includes/vendor/*'
+PHPCS_RULESET_FILE=./.phpcs.xml.dist
```

Comment:
GitHub dropped my PR comment. I think it would be better to name this just `phpcs.xml.dist` so that `phpcs` can be run from the directory without having to pass the `--standard`.

Reply:
Actually, GH didn't drop your comment, but published it as a stand-alone comment. See my response above.